controllers/machinedeletionremediation_controller_test.go

```diff
@@ -25,6 +25,7 @@ import (
 
 const (
 	defaultNamespace                                                          = "default"
+	machineNamespace                                                          = "openshift-machine-api"
```

Comment:
NIT: I think it makes more sense to declare this const in `controllers/suite_test.go` or even in a different package/file as global const since this namespace is probably needed in different places (e.g., e2e tests).

Reply:
I'm not sure. e2e tests only test manually created CR, so the OwnerReference is always empty and there is no need for this Namespace